### PR TITLE
Reinstate the back to SP link

### PR DESCRIPTION
**Pull request by MKodde**
The back to SP link was added to the base template.

Todo:
 - The `focus` style shows the border with to much margin, causing a not so nice render on the screen. @Badlapje can you help?
 - Tests for this feature should be added

### CODE REVIEW

**Comment by Badlapje** [comment]
on it ;-)

File: theme/base/templates/modules/Authentication/View/Proxy/Partials/WAYF/backButton.html.twig [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{% if backLink is defined and backLink == true %}
```
Comment (Badlapje) [inline theme/base/templates/modules/Authentication/View/Proxy/Partials/WAYF/backButton.html.twig:1]: shouldn't this file be called `backLink.html.twig`?  And the class `wayf__backLink`?  I'm fine either way though.

**Review by Badlapje — COMMENTED** [review]
Looking good 👍 

One question though: why was NunitoOFL.txt removed?  License info like that should not go missing imo.  We could add it as a comment to fonts.scss?

If we do remove it, then probably we should delete `-e base/stylesheets/fonts/*.txt` from the build:copy script in package.json.

File: theme/base/templates/modules/Authentication/View/Proxy/Partials/WAYF/backButton.html.twig (hunk 1 above)
Comment (MKodde) [reply theme/base/templates/modules/Authentication/View/Proxy/Partials/WAYF/backButton.html.twig:1]: Very sharp, seeing this is a link and not a button! Will change that

**Comment by MKodde** [comment]
> One question though: why was NunitoOFL.txt removed? License info like that should not go missing imo. We could add it as a comment to fonts.scss?

Personally I'm fine with keeping the file, but then we must update the git ignore file to prevent adding it unintentionally. Also we should think/discuss on how we should go about using these fonts and how we'd like to advertise we are using this font. Maybe your comment is a good proposal. Let's discuss this in the stand up or on the Slack channel

**Review by Badlapje — APPROVED** [review]
Nice! 💯

**Pull request merged**